PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml

```diff
@@ -68,15 +69,19 @@
                 android:layout_weight="1"
                 android:background="@drawable/vocab_clipboard_yellow"
                 android:text="@string/bra"
-                android:gravity="center"/>
+                android:gravity="center"
+                android:paddingLeft="@dimen/vocam_match_padding"
+                />
             <powerup.systers.com.vocab_match_game.VocabBoardTextView
                 android:id="@+id/tv3"
                 android:layout_width="match_parent"
                 android:layout_height="0dp"
                 android:layout_weight="1"
                 android:background="@drawable/vocab_clipboard_yellow"
                 android:text="@string/periods"
-                android:gravity="center"/>
+                android:gravity="center"
+                android:paddingLeft="@dimen/vocam_match_padding"
+                />
```

Comment:
Write line no 83 in the following way: ```android:paddingLeft="@dimen/vocam_match_padding" />```
Always try to avoid the use of extra lines.